twisted/conch/ssh/forwarding.py

```diff
@@ -185,9 +185,11 @@ def connectionLost(self, reason):
             self.channel = None
 
 
-def packOpen_direct_tcpip((connHost, connPort), (origHost, origPort)):
+def packOpen_direct_tcpip(conn_host_port, orig_host_port):
```

Comment:
conn_host_port -> destinationi
orig_host_port -> source

and we need a docstring for this, to declare the type and format of these arguments